metaflow/plugins/kfp/tests/.gitlab-ci.yml

```diff
@@ -201,8 +202,9 @@ test:prod:
           export KFP_RUN_URL_PREFIX=https://kubeflow.corp.prod-k8s.zg-aip.net/ && 
           export KFP_SDK_NAMESPACE=metaflow-integration-testing-prod && 
           export USER=${GITLAB_USER_EMAIL} &&
+          export METAFLOW_DATASTORE_SYSROOT_S3=s3://serve-datalake-zillowgroup/zillow/workflow_sdk/metaflow_28d/prod/aip-integration-testing
```

Comment:
These env variables are set in KF Profile, see http://gitlab.zgtools.net/analytics/artificial-intelligence/ai-platform/aip-infrastructure/kubeflow-k8s-profiles/blob/8e88dcca3dbba7383393fb291d3d4a933cb2d132/templates/all-pod-default.yaml#L68-L68

Reply:
Yup, but unfortunately, they're not present within the Gitlab runners :( If I run as is without passing them here, the flow doesn't compile because those env variable aren't recoginized.

Reply:
We were able to get away with this earlier because the flows currently are launched without an S3 code package, so the datastore root was not used at compile time.

Reply:
These test flows are also not using an s3 code package, they are using: `--no-s3-code-package --base-image`

Reply:
True, but within the flow files, we access the env var like `environ["METAFLOW_DATASTORE_SYSROOT_S3"]`. Since that env var isn't available on the Gitlab runners, Metaflow complains during the compilation phase.

Reply:
Can you have it evaluate the env variable at runtime?